src/Async/EventStoreCatchUpSubscription.php

```diff
@@ -29,6 +28,4 @@ public function subscriptionName(): string;
     public function startAsync(): Promise;
```

Comment:
what about this here? basically this is an implementation detail, how to start it and it should not get called by the end user.

Reply:
Removed it both here and on the persistent subscription interface.